.env.test

```diff
@@ -19,6 +19,7 @@ SQS_DEPLOYMENT_QUEUE_REGION=us-west-1
 SQS_BRANCH_QUEUE_URL=http://127.0.0.1:4566/000000000000/test-branch
 SQS_BRANCH_QUEUE_REGION=us-west-1
 BULL_QUEUE_PREFIX=bulltest
+HASH_SECRET=test
```

Comment:
I feel like we should rename this to be more explicit, something like `GLOBAL_HASH_KEY`